olive/common/utils.py

```diff
@@ -19,8 +19,8 @@
 logger = logging.getLogger(__name__)
 
 
-def run_subprocess(cmd, env=None, cwd=None, check=False):  # pragma: no cover
-    logger.debug(f"Running command: {cmd} with env: {env}")
```

Comment:
encode the env to UTF-8 in case of error happens in line 40?

Reply:
Updated.